lib/charms/layer/jenkins/api.py

```diff
@@ -167,6 +171,14 @@ def get_node_secret(self, node_name):
         except jenkins.JenkinsException:
             return False
 
+    def set_update_center(self, url=None, default=False):
+        """Set the update center or reset it to default"""
+        if default:
+           url = "https://updates.jenkins.io/stable/update-center.json"
```

Comment:
This seems a bit odd - if we set `default = True` we're overriding whatever we've passed for `url`. Why not just have `url="https://updates.jenkins.io/stable/update-center.json"` in the function definition and allow that to be changed to avoid the need for the `default` argument?

Reply:
Overthinking on my part. Fixed in: e88dad6